### PR TITLE
Add documentation about running LMS tests.

### PR DESCRIPTION
**Pull request by vkaracic**
@edx/docker-devstack-working-group please review.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -306,6 +306,17 @@ Start over
 If you want to completely start over, run ``make destroy``. This will remove
 all containers, networks, AND data volumes.
 
+Run LMS tests
```
Comment (robrap) [inline README.rst]: 1. Rename to `LMS and Studio Python tests`.
2. I would move this as a higher-level section immediately before or after PyCharm Integration, to hopefully keep people from having trouble. :)
3. Add another line at the end: `You can also add the ``test_docker`` settings flag to the other examples detailed in the edx-platform testing documentation.`, making `edx-platform testing documentation` a reference to https://github.com/edx/edx-platform/blob/master/docs/testing.rst#running-python-unit-tests

File: README.rst [hunk 2]
```diff
@@ -306,6 +306,17 @@ Start over
...
+~~~~~~~~~~~~~
+
+If you want to run tests inside the LMS container, you need to pass the
```
Comment (robrap) [inline README.rst]: `tests` => `python tests`

**Review by robrap — COMMENTED** [review]
Thanks for adding this!

FYI: @macdiesel: as part of the edx-platform workflow docs work it may make sense to pull this out to another doc, but I think this is fine here until that larger work gets started.

**Comment by vkaracic** [comment]
@robrap no problem! I believe I have addressed your comments. Please re-review.

**Review by robrap — APPROVED** [review]

**Pull request merged**